src/controller/python/ChipDeviceController-InteractionModelDelegate.h

```diff
@@ -0,0 +1,72 @@
+#include <app/InteractionModelDelegate.h>
+
+namespace chip {
+namespace Controller {
+
+struct __attribute__((packed)) CommandStatus
```

Comment:
Consider adding a comment that packed is used to make python binary data generation easy (not worry about alignment)